R/mod_category_criticality.R

```diff
@@ -24,6 +24,9 @@ mod_category_criticality_ui <- function(id){
       tabsetPanel(
```

Comment:
OK for now but let's please not have lots of commented code in pull requests- the old code is in the git history so if we're deleting code, let's just delete it properly